doc/script_commands.txt

```diff
@@ -5622,6 +5625,30 @@ Examples:
 
 ---------------------------------------
 
+*equipidx(<index>)
+
+The equipidx function will attempt to equip the item at the given inventory
+inventory index. This feature is only really useful when multiple instances
+of an Item ID exist in the player's inventory.
+
+Returns 1 if the item was successfully equipped.
+Returns 0 if the item failed to equip.
+
+See getinventorylist() for an explanation on retrieving an inventory index.
+
+Examples:
+
+//	This will prioritize equip a rental Knife
+	getinventorylist();
+	for (.@i = 0; .@i < @inventorylist_count; ++.@i) {
+		if (@inventorylist_id[.@i] == Knife && @inventorylist_expire[.@i]) {
+			equipidx(.@i);
```

Comment:
thanks to dastgir idea, that just push another array of the index,
that means this example has to change accordingly

```suggestion
 		equipidx(@inventorylist_idx[.@i]);
```